serving/docker/partition/trt_llm_partition.py

```diff
@@ -27,8 +27,9 @@ def create_trt_llm_repo(properties, args):
             kwargs[key[7:]] = value
         else:
             kwargs[key] = value
-    kwargs['trt_llm_model_repo'] = args.trt_llm_model_repo
     kwargs = update_kwargs_with_env_vars(kwargs)
+    kwargs['trt_llm_model_repo'] = args.trt_llm_model_repo
+    kwargs["tensor_parallel_degree"] = args.tensor_parallel_degree
     model_id_or_path = args.model_path or kwargs['model_id']
     if 'max' == kwargs.get('tensor_parallel_degree', -1):
```

Comment:
this can be removed as it's handled on the java side. Also args.gpu_count is invalid as it's been changed to `tensor_parallel_degree`

Reply:
Fixed